src/sentry/static/sentry/app/components/events/groupingInfo.jsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import _ from 'lodash';
```

Comment:
afaik It would actually be better to import as:

```jsx
import {isObject} from 'lodash'
```

So the tree shaker can better drop lodash stuff

Reply:
Changed. I also just copied that pattern from elsewhere :)